pkg/auth/cmd.go

```diff
@@ -2,32 +2,33 @@ package auth
 
 import (
 	"github.com/spf13/cobra"
+	"io/ioutil"
 )
 
 func LoginCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "login",
 		Short: "Login",
+		Long: `Log a user in using its Console credentials and save the login token to disk, 
+to allow the CLI access to the STRM Privacy APIs.`,
 		Run: func(cmd *cobra.Command, args []string) {
 			login()
 		},
-		Args: cobra.ExactArgs(0),
+		DisableAutoGenTag: true,
+		Args:              cobra.ExactArgs(0),
 	}
 	return cmd
 }
-
 func PrintTokenCmd() *cobra.Command {
+	var content, _ = ioutil.ReadFile("pkg/auth/docstring_print_token.md")
```

Comment:
Nice, reading it from file! But, does the CLI still run? Because you'll read this file every time the print token command is executed.

Reply:
yes it does, as the content is just an empty string if its not available. However, might be better to declare as variable outside of the function